src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/MsBuild/MsBuildTempProjectFile.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Build.Construction;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    public class MsBuildTempProjectFile : IDisposable
```

Comment:
Nit: `MsBuildProjectFile` is fine too since this is already part of the UnitTests